swinging-cat-rx/src/animation.js

```diff
@@ -0,0 +1,116 @@
+import {Observable} from 'rxjs'
+
+/* CONSTATNST */
+
+const TRANSFORM = 'transform'
```

Comment:
In JS we don't need constants for strings, because we have no refs to strings, it is always a copy.